src/theme/theme.stories.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import withReadme from 'storybook-readme/with-readme'
+import styled from 'styled-components'
+import Readme from './Readme.md'
+import { H3, Body2, Caption } from '../components/typography'
+
+const sharedStyle = `
+  color: rgba(255, 255, 255, .87);
+  padding: 60px;
+  text-align: center;
+  flex-grow: 1;
+  flex-basis: 0;
+  border: 1px solid white;
+`
+
+const BrandBasePalette = styled.div`
+  background-color: ${props => props.theme.color('brand')};
+  ${sharedStyle}
+`
+
+const PrimaryBasePalette = styled.div`
+  background-color: ${props => props.theme.color()};
+  ${sharedStyle}
+`
+
+const SecondaryDarkPalette = styled.div`
+  background-color: ${props => props.theme.color('secondary', 'dark')};
+  ${sharedStyle}
+`
+
+const SecondaryBasePalette = styled.div`
+  background-color: ${props => props.theme.color('secondary', 'base')};
+  ${sharedStyle}
+  color: rgba(0, 0, 0, .7);
+`
+
+const SecondaryLightPalette = styled.div`
+  background-color: ${props => props.theme.color('secondary', 'light')};
+  ${sharedStyle}
+  color: rgba(0, 0, 0, .7);
+`
+
+const SecondaryBrightPalette = styled.div`
+  background-color: ${props => props.theme.color('secondary', 'bright')};
+  ${sharedStyle}
+  color: rgba(0, 0, 0, .7);
+`
+
+const NeutralDarkPalette = styled.div`
+  background-color: ${props => props.theme.color('neutral', 'dark')};
+  ${sharedStyle}
+`
+
+const NeutralBasePalette = styled.div`
+  background-color: ${props => props.theme.color('neutral')};
+  ${sharedStyle}
+`
+
+const NeutralBrightPalette = styled.div`
+  background-color: ${props => props.theme.color('neutral', 'bright')};
+  ${sharedStyle}
+  color: rgba(0, 0, 0, .7);
```

Comment:
Should these colors also be variables?

Reply:
we already have in fact... Fixed it to use `theme.contrast` function

Reply:
👍